tests/actions.go

```diff
@@ -3697,6 +3698,47 @@ func (oa *OperatorActions) WaitForTidbClusterReady(tc *v1alpha1.TidbCluster, tim
 	return err
 }
 
+func (oa *OperatorActions) WaitForConnectTidbCluster(tc *v1alpha1.TidbCluster, timeout, pollInterval time.Duration) error {
```

Comment:
this method only works for TC with `PasswordInitialized` right? Can we give it a meaningful name?

Reply:
ok